masq_lib/src/shared_schema.rs

```diff
@@ -526,9 +526,110 @@ impl ConfiguratorError {
+     zero-hop means that your Node will operate as its own MASQ Network and will not communicate with any \
+     other Nodes. --ip, --neighbors, and --clandestine-port are incompatible with --neighborhood_mode \
+     zero-hop.\n\n\
+     originate-only means that your Node will not accept connections from any other Node; it \
+     will only originate connections to other Nodes. This will reduce your Node's opportunity to route \
+     data (it will only ever have two neighbors, so the number of routes it can participate in is limited), \
+     it will reduce redundancy in the MASQ Network, and it will prevent your Node from acting as \
+     a connection point for other Nodes to get on the Network; but it will enable your Node to operate in \
+     an environment where your network hookup is preventing you from accepting connections, and it means \
+     that you don't have to forward any incoming ports through your router. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode originate-only.\n\n\
+     consume-only means that your Node will not accept connections from or route data for any other Node; \
+     it will only consume services from the MASQ Network. This mode is appropriate for devices that \
+     cannot maintain a constant IP address or stay constantly on the Network. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode consume-only.\n\n\
+     standard means that your Node will operate fully unconstrained, both originating and accepting \
+     connections, both consuming and providing services, and when you operate behind a router, it \
+     requires that you forward your clandestine port through that router to your Node's machine.");
+        assert_eq!(MAPPING_PROTOCOL_HELP, "The Node can speak three protocols to your router to make it allow outside Nodes to connect inward \
+    through it to your machine. These three protocols are pcp, pmp, and igdp. The Node can try them one \
+    by one to determine which your router supports, but if you happen to know already, you can supply the \
+    name of the protocol here. If you've taken care of port mapping in some other way, \
+    and you don't need Node to negotiate with your router, say 'none' here and be sure to specify your \
+    public IP address with the --ip parameter. If the Node communicates successfully with your router, \
+    it will remember the protocol it used, and on its next run it will try that protocol first, unless \
+    you specify a different protocol on the command line.");
+        assert_eq!(REAL_USER_HELP,"The user whose identity Node will assume when dropping privileges after bootstrapping. Since Node refuses to \
+     run with root privilege after bootstrapping, you might want to use this if you start the Node as root, or if \
+     you start the Node using pkexec or some other method that doesn't populate the SUDO_xxx variables. Use a value \
+     like <uid>:<gid>:<home directory>.");
+    }
+
```

Comment:
There is a place in `masq/src/schema.rs` where we're configuring the Clap's App object. We write the "author' in and also the version, probably a line of description too. You could bring in new constants for each of them and then also test them the way you test constants at other places. 

I'd like you to do the same with Clap code in `masq/src/commands.rs` where probably every file in there (except command_common.rs) contains a description - which is not tested in fact. 

And also in node/src/apps.rs. ;)

Reply:
@bertllll 
I would suggest you to have a look at this particular commit - https://github.com/MASQ-Project/Node/commit/e5cabf9de5bf7270b492ece00292103fb0c54b89

Here, I created way too many constants, do you think it is a good decision to create so many constants and then test them?